automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSpec.scala

```diff
@@ -314,42 +314,5 @@ class LeonardoSpec extends FreeSpec with Matchers with Eventually with ParallelT
       val googlePetEmail2 = googleIamDAO.findServiceAccount(project, petName).futureValue.map(_.email)
       googlePetEmail2 shouldBe Some(samPetEmail)
     }
-
-    // TODO retrieving service account keys from Google is SUPER inconsistent and flakey
```

Comment:
This test is kind of obviated since we're no longer managing pet SA keys in Leo.

Will push a follow-on commit with additional swat tests.